code/modules/rigsuits/rig.dm

```diff
@@ -83,19 +83,19 @@
 	var/datum/effect_system/spark_spread/spark_system
 
 /obj/item/rig/examine()
-	//to_chat(usr, "This is [bicon(src)][src.name].")
-	to_chat(usr, "This is [src.name].")
-	to_chat(usr, "[src.desc]")
+	//. += "This is [bicon(src)][src.name]."
```

Comment:
`.` will be `null` by default and this will runtime, won't it?
Needs to call parent first.